magpie/adapter/magpieowssecurity.py

```diff
@@ -155,7 +162,8 @@ def check_request(self, request):
                                  service_impl.service.resource_name, type(perm_exc).__name__, perm_exc)
 
             if permission_requested:
-                LOGGER.info("'%s' request '%s' permission on '%s'", request.user, permission_requested, request.path)
+                LOGGER.info("User %s is requesting '%s' permission on [%s]",
```

Comment:
Is Twitcher and Magpie container python 3.6 and newer?  If yes, you can use f-string that is way more readable.

That log line with f-string will look like `LOGGER.info(f"User {request.user} is requesting '{permission_requested}' permission on [{request.path}]"`

Reply:
It's always better to have `%s` with extra args in logging calls because it skips the formatting step entirely when log level is not reached. This adds-up quickly in term of delays if there are a lot of log calls.

Reply:
Oh nice, I have not thought about this performance optimization side.